packages/material-ui-styles/README.md

```diff
@@ -8,10 +8,10 @@ Install the package in your project directory with:
 
 ```sh
 // with npm
-npm install @material-ui/styles
+npm install @material-ui/styles @emotion/core @emotion/styled
 
 // with yarn
-yarn add @material-ui/styles
+yarn add @material-ui/styles @emotion/core @emotion/styled
```

Comment:
Does the styles package really need these new dependencies? I was under the assumption that we would try not to interfer with it, progressively making it legacy, and dropping it.

Reply:
The only reason there were added is because the `ThemeProvider` is in this package. Once we move it out of the package we can drop these dependencies

Reply:
We only document the import of `ThemeProvider` from the core. Would it work if we moved the theme provider nesting to the core?

My concern is about the migration experience for v4 developers that are using JSS. We could imagine having developers migrate their v4 imports from `@material-ui/core/styles` to  `@material-ui/styles` and have still everything working. We could no longer have @material-ui/styles as a dependency in @material-ui/core. Would that work? Do you have an alternative story in mind? A migration from `@material-ui/core/styles` to react-jss won't be as smooth.

Reply:
Ah I see what is the concern. Let me try then to migrate the nesting to the core 👍

Reply:
Done 👍